CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # SourceGrid
 
-## Major Changes
+## [1.0.0] - 2019-04-30
```

Comment:
one last change..add the new changes on top.. else we have to scroll down to read :)

Reply:
Done